src/main/java/org/example/results/Aggregate.java

```diff
@@ -18,7 +18,7 @@ public class Aggregate {
     @Id
     @GeneratedValue
     @Column(name = "_id")
-    private Integer surveyID;
+    private Integer aggregateID;
```

Comment:
I think we should also have a surveyID attribute still so it can be referenced if the user wants to choose an aggregate

Reply:
In that case, it might just be better to add a reference to the aggregateID in the survey class instead, it would be a one-to-one relationship and we'd use the survey to get to the aggregate...

Let me know what you think! I appreciate the suggestion!

Reply:
That could work too but I think if for whatever reason the aggregate is remade, it would be easier for the aggregate to know which survey it is for than the other way around.

Reply:
Good point...
But it's not expected that we ever remake an aggregate as the compile function is only ever called one time (when the survey is closed). The compile class (which is the only way to make an aggregate) also gets passed the Survey anyways so the association between the survey and the questions will remain at all times. 

Again let me know what you think! Open to suggestions!

Reply:
Let's see what the others think and go from there.

Reply:
I think I agree with Momo here, having a reference to the aggregate from the survey sounds better since the survey is the centre of our application, and would probably make design easier if we keep it as such.
We can create a field in the survey that is populated by the manager when we compile, and when we want to retrieve the aggregate we provide the surveyID, parse the arrayList in manager, and retrieve the aggregate from the survey object.

Reply:
@momokaddour Can you make this change in the PR?